### PR TITLE
Deprecate software 0.29 and 0.28

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/1961
Resolves https://github.com/astronomer/docs/issues/1960

### CODE REVIEW

File: software_versioned_docs/version-0.30/version-compatibility-reference.md [hunk 1]
```diff
@@ -15,19 +15,15 @@ While the tables below reference the minimum compatible versions, we typically r
 
 | Astronomer Platform | Kubernetes                           | Postgres | Python                                    | Astronomer Certified / Astro Runtime         | Helm |
```
Comment (jwitz) [inline software_versioned_docs/version-0.30/version-compatibility-reference.md:16]: @shmanu017 Can you confirm that this is up to date?

File: software_versioned_docs/version-0.30/version-compatibility-reference.md [hunk 2]
```diff
@@ -15,19 +15,15 @@ While the tables below reference the minimum compatible versions, we typically r
...
 | ------------------- | ------------------------------------ | -------- | ----------------------------------------- | -------------------------------------------- | ---- |
-| v0.28               | 1.19¹, 1.20¹, 1.21, 1.22, 1.23, 1.24 | 9.6+     | 3.6, 3.7, 3.8, 3.9 (_requires AC 2.2.0+_) | All supported Certified versions             | 3.6  |
-| v0.29               | 1.19¹, 1.20¹, 1.21, 1.22, 1.23, 1.24 | 9.6+     | 3.6, 3.7, 3.8, 3.9 (_requires AC 2.2.0+_) | All supported Certified and Runtime versions | 3.6  |
 | v0.30               | 1.19¹, 1.20¹, 1.21, 1.22, 1.23, 1.24 | 9.6+     | 3.6, 3.7, 3.8, 3.9 (_requires AC 2.2.0+_) | All supported Certified and Runtime versions | 3.6  |
```
Comment (shmanu017) [inline software_versioned_docs/version-0.30/version-compatibility-reference.md:18]: Supported k8s version should include 1.25 and 1.24 as supported versions @jwitz

**Comment by kwcanuck** [comment]
@jwitz the links to the 0.28 and 029 versions in the _[Astronomer Software documentation archive](https://deploy-preview-1962--gallant-neumann-56599d.netlify.app/software/documentation-archive)_ topic return 404 errors. I assume this is expected as the repository doesn't exist yet?

**Review by kwcanuck — APPROVED** [review]

**Comment by shmanu017** [comment]
Please add a Note : Install/Update to recommended Postgres version 11.19+ for software setup [ref](https://endoflife.date/postgresql)

**Pull request merged**